js/data/apply_changes.js

```diff
@@ -0,0 +1,3 @@
+import { applyChanges } from './array_utils';
```

Comment:
Why don't you isolate the applyChanges and applyBatch functions in this file instead of placing them in array_utils?

Reply:
With current exports we have two internal sources of applyChanges.

Reply:
> Why don't you isolate the applyChanges and applyBatch functions in this file instead of placing them in array_utils?


I do not think that the applyBatch method should be moved to apply_changes. The method is intended to arrays and should be placed in array_utils as it is. 

> With current exports we have two internal sources of applyChanges.

We have many places not only in grid where we re-export API and there is no problem with it.